test/unit/strings-test.js

```diff
@@ -0,0 +1,59 @@
+define([
+    'utils/strings'
+], function (strings) {
+    /* jshint qunit: true */
+
+    module('strings');
+
+    test('strings.pad', function(assert) {
+        var str = strings.pad('test', 7, '1');
+
+        assert.equal(str, '111test', 'strings padding correctly done');
```

Comment:
Let's also test when the lend of the 1st param is >= the second param (no padding added)